src/components/Select/Select/Select.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from 'react';
+import { useSelect } from 'downshift';
+import { IoMdArrowDropup, IoMdArrowDropdown } from 'react-icons/io';
+import { bem } from '../../../utils/bem/bem';
+import { FieldWrapper } from '../../FieldWrapper';
+import { List } from '../../List';
+import { BasicSelectProps } from '../SelectBase';
+import { SuggestionsList } from '../SuggestionsList';
+import styles from './Select.scss';
+
+const { block, elem } = bem('Select', styles);
+
+interface Props<S> extends Omit<BasicSelectProps<S>, 'isProminent'> {
+    /** The suggestions that is currently selected */
+    selectedSuggestion: S;
+}
+
+export function Select<S>(props: Props<S>) {
+    const {
+        suggestions,
+        selectedSuggestion,
+        suggestionToString,
+        suggestionItemRenderer,
+        onSelectionAdd,
+        onBlur,
+        onFocus,
+        rootRef,
+        listRef,
+        ...rest
+    } = props;
+
+    const handleSelection = (state) => {
+        onSelectionAdd(state.selectedItem);
+        onBlur?.();
+    };
+
+    const {
+        isOpen,
+        getToggleButtonProps,
+        getMenuProps,
+        highlightedIndex,
+        getItemProps,
+    } = useSelect({
+        items: suggestions,
+        selectedItem: selectedSuggestion,
+        onSelectedItemChange: handleSelection,
+    });
+
+    const handleToggle = (e) => {
+        e?.stopPropagation();
+        if (isOpen) {
+            onBlur?.();
+        } else {
+            onFocus?.();
+        }
+    };
+
+    return (
+        <div ref={rootRef} {...rest} {...block({})}>
+            <div {...elem('main', { isOpen })}>
+                <FieldWrapper isFocused={isOpen} {...elem('field', {})}>
```

Comment:
It's not really necessary to pass an empty object to `elem` and `block` functions as a second parameter so it can be omitted.

Reply:
good catch. removed